<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>683</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>611</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background:rgb(248, 248, 248);
border-radius: 5%;
color:rgb(75, 75, 75)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>591</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_16">
      <item>
       <widget class="QLabel" name="label_23">
        <property name="styleSheet">
         <string notr="true">background:rgb(228, 246, 248);
color: rgb(75, 75, 75);
font: 800 10pt &quot;Trebuchet MS&quot;;
padding: 5;
border-radius: 0%</string>
        </property>
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <item>
           <widget class="QLabel" name="label_24">
            <property name="styleSheet">
             <string notr="true">font: 75 9pt &quot;Trebuchet MS&quot;;</string>
            </property>
            <property name="text">
             <string>Search by Name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_11">
            <property name="leftMargin">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>15</number>
            </property>
            <property name="rightMargin">
             <number>15</number>
            </property>
            <property name="bottomMargin">
             <number>15</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Last Name:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="search_last">
              <property name="styleSheet">
               <string notr="true">background: white;
border-radius: 0%;</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>First Name:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="search_first">
              <property name="styleSheet">
               <string notr="true">background: white;
border-radius: 0%;</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background:rgb(217, 217, 217)</string>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_18">
          <item>
           <widget class="QLabel" name="label_27">
            <property name="styleSheet">
             <string notr="true">font: 75 9pt &quot;Trebuchet MS&quot;;
padding-top: 5</string>
            </property>
            <property name="text">
             <string>Search by Donor Number</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_12">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="leftMargin">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>15</number>
            </property>
            <property name="rightMargin">
             <number>15</number>
            </property>
            <property name="bottomMargin">
             <number>15</number>
            </property>
            <item row="0" column="0">
             <widget class="Line" name="line_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="donorNumberLabel_6">
              <property name="text">
               <string>Donor Number:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="search_num">
              <property name="styleSheet">
               <string notr="true">background: white;
border-radius: 0%;</string>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="enterManuallyLabel_6">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="search_validate">
        <property name="styleSheet">
         <string notr="true">color: rgb(249, 66, 58);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="search">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;
background: rgb(85, 198, 229);
border-radius: 5%;
font: 700 10pt &quot;Trebuchet MS&quot;;
padding-top: 5;
padding-bottom: 5;
padding-left: 12;
padding-right: 12;
color: white;
letter-spacing: .2rem
</string>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
          <property name="iconSize">
           <size>
            <width>12</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>611</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(75, 75, 75);
background:rgb(248, 248, 248);
border-radius: 5%;
border: solid black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>591</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QLabel" name="label_19">
        <property name="styleSheet">
         <string notr="true">background:rgb(228, 246, 248);
color: rgb(75, 75, 75);
font: 800 10pt &quot;Trebuchet MS&quot;;
padding: 5;
border-radius: 0%</string>
        </property>
        <property name="text">
         <string>Add Donor: </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <layout class="QFormLayout" name="formLayout_9">
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>15</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Last Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="last">
            <property name="styleSheet">
             <string notr="true">background: white;
border-radius: 0%;</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>First Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="first">
            <property name="styleSheet">
             <string notr="true">background: white;
border-radius: 0%;</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="dOBLabel_3">
            <property name="text">
             <string>DOB:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="dob">
            <property name="whatsThis">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white;
border-radius: 0%;</string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>MM-DD-YYYY</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="donorNumberLabel_5">
            <property name="text">
             <string>Donor Number:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="num">
            <property name="styleSheet">
             <string notr="true">background: white;
border-radius: 0%;</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="treeLevelLabel_4">
            <property name="text">
             <string>Number of Donations:</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="donations">
            <property name="styleSheet">
             <string notr="true">background: white;
border-radius: 0%;</string>
            </property>
            <property name="currentText">
             <string>Choose Number</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>Choose Number</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>300</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>700</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>900</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1000</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1100</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1300</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1500</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1700</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1900</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2000</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>(all fields required)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <layout class="QFormLayout" name="formLayout_10">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="leftMargin">
             <number>15</number>
            </property>
            <property name="topMargin">
             <number>15</number>
            </property>
            <property name="rightMargin">
             <number>15</number>
            </property>
            <property name="bottomMargin">
             <number>15</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="centerLocationLabel_7">
              <property name="text">
               <string>Center Location:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="center">
              <property name="styleSheet">
               <string notr="true">background: white;
border-radius: 0%;</string>
              </property>
              <item>
               <property name="text">
                <string>Choose Location</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Burns</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Portland</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Reno</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Seattle</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="branchNumberLabel_3">
                <property name="text">
                 <string>Branch Number:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QLineEdit" name="branch">
                <property name="styleSheet">
                 <string notr="true">background: white;
border-radius: 0%;</string>
                </property>
                <property name="frame">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>1</number>
              </property>
              <item>
               <widget class="QPushButton" name="findbranch">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;
font: 8pt &quot;Trebuchet MS&quot;;
background: white;
border-radius: 5%;

color:rgb(75, 75, 75);
padding: 7;
</string>
                </property>
                <property name="text">
                 <string>Find Branch</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="info">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">border: none;
background:none;
border-radius: 30%;
font: 800 12pt &quot;Trebuchet MS&quot;;
padding-top: 5;
padding-bottom: 5;
padding-left: 15;
padding-right: 15;
color: rgb(100, 105, 112);

text-align: left;</string>
                </property>
                <property name="text">
                 <string>?</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="enterManuallyLabel_5">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="validate">
              <property name="styleSheet">
               <string notr="true">color:rgb(247, 65, 58)</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <spacer name="horizontalSpacer_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="add">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;
background: rgb(85, 198, 229);
border-radius: 5%;
font: 700 10pt &quot;Trebuchet MS&quot;;
padding-top: 5;
padding-bottom: 5;
padding-left: 15;
padding-right: 15;
color: white;
letter-spacing: .2rem
</string>
          </property>
          <property name="text">
           <string>Add Donor</string>
          </property>
          <property name="iconSize">
           <size>
            <width>12</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background:none;
color: white</string>
    </property>
    <property name="text">
     <string>✖</string>
    </property>
   </widget>
   <widget class="QLabel" name="banner">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(159, 209, 155, 200); 
font: 75 8pt &quot;Trebuchet MS&quot;;
color: white;
padding-left: 15;
padding-top:5;
padding-bottom:5;
border-radius: 10%;</string>
    </property>
    <property name="text">
     <string>Donor removed successfully.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="close_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background:none;
color: white</string>
    </property>
    <property name="text">
     <string>✖</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
